app/controllers/appeals_base_controller.rb

```diff
@@ -36,4 +43,38 @@ def get_hash_from_request_body
   def request_body_is_not_a_hash_error
     DecisionReview::ServiceException.new key: 'DR_REQUEST_BODY_IS_NOT_A_HASH'
   end
+
+  def request_body_debug_data
```

Comment:
Can these methods that you introduced here be moved out to a helper or concern to avoid controller bloating?

Reply:
@LindseySaari  I'm down for that! Do you have any suggestions on a good name? I'm not so great at naming Concerns.

Reply:
Trying to think of how to categorize these new methods? Maybe something like Appeals Data? Or even AppealsBaseConcern might work?

Reply:
most of these methods are for helping log a failed request to the PII table.
`FailedRequestLoggable`
`PersonalInformationLoggable`
`PiiLoggable`
?

Reply:
Ooo I like all of those. Maybe FailedRequestLoggable? You're name suggestions were WAY better than mine 😺